### ISSUE TITLE
Issue: Outlined icons have #fff paths set on them

### ISSUE THREAD

**Issue by ghost** [graywolftech/react-heroicons#6]
Example: AcademicCapOutline

When I add this in my react project from this package it results in:
```
<svg class="text-purple-300 mr-3 h-6 w-6" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
  <path fill="#fff" d="...."></path>
  <path fill="#fff" d="..."></path>
</svg>
```
This path fill is something that cannot be altered from just referencing `<AcademicCapOutline />`
Perhaps you can remove the path fills alltogether (as heroicons.dev does)?

**Comment by jsmith** [comment]
This is a duplicate of https://github.com/tailwindlabs/heroicons/issues/93 (which I think you already know). I can probably add a simple find and replace to remove the fill colors though.

**Comment by jsmith** [comment]
Closed by 5a5e2e40226accf68b92e900c2eaf4162ba63e59
